include/swift/Basic/LangOptions.h

```diff
@@ -305,6 +305,10 @@ namespace swift {
     /// set to true.
     bool ExperimentalDependenciesIncludeIntrafileOnes = false;
 
+    /// We disallow explicit dynamic on native swift declarations in
+    /// library-evolution-mode for now because the ABI is not fully baked.
+    bool DisallowDynamicOnNativeSwiftDeclarations = false;
```

Comment:
This doesn't really make sense to me; it's not something we'd turn on and off with a flag. I think it's okay to just check for EnableLibraryEvolution directly.

Reply:
I did it this way because EnableLibraryEvolution is on FrontendFlags which I believed to be not available in Sema.

Reply:
Oh, oops. It is, but you can get that information from the containing module. `ModuleDecl::getResilienceStrategy`.

Reply:
Oh right. I will do so.